src/WebJobs.Extensions.DocumentDB/Properties/AssemblyInfo.cs

```diff
@@ -1,17 +1,16 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Reflection;
 using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
 
 [assembly: AssemblyProduct("Microsoft Azure WebJobs SDK DocumentDB Extensions")]
-[assembly: AssemblyCompany("Microsoft Corporation")]
-[assembly: AssemblyCopyright("© Microsoft Corporation. All rights reserved.")]
+
 [assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
Should be 1.2.0 (to match the fact that we're at package version 1.2.0-beta now)? Just like in the SDK repo, I think I forgot these